### ISSUE TITLE
Compatibility with Thumbor 7.0.0

### ISSUE THREAD

**Issue by heynemann** [Superbalist/thumbor-cloud-storage#10]
Hey, how can I help you add compatibility to thumbor 7?

**Comment by heynemann** [comment]
For more information on what changed you can check the release at https://github.com/thumbor/thumbor/releases/tag/7.0.0

**Comment by heynemann** [comment]
It would be awesome if the community could use this extension.

**Comment by leonardo2204** [comment]
Agreed, @heynemann. But I'm not sure if this extensions is maintained :(
